roles/kubernetes-apps/ingress_controller/cert_manager/templates/cert-manager.yml.j2

```diff
@@ -939,6 +951,18 @@ spec:
                 fieldPath: metadata.namespace
           resources:
             {}
+{% if cert_manager_tolerations %}
+      tolerations:
+        {{ cert_manager_tolerations | to_nice_yaml(indent=2) | indent(width=8) }}
+{% endif %}
+{% if cert_manager_nodeselector %}
+      nodeSelector:
+        {{ cert_manager_nodeselector | to_nice_yaml | indent(width=8) }}
+{% endif %}
+{% if cert_manager_affinity %}
+      affinity:
+        {{ cert_manager_affinity | to_nice_yaml | indent(width=8) }}
+{% endif %}
```

Comment:
Seems like this block needs to be moved below the next block, since using both `cert_manager_trusted_internal_ca` and one of these 3 options causes `volumeMounts` to become a key under the wrong branch (it should be under `containers` above).

Just ran into this issue, thought I'd post it here in case anyone wants to check or fix it.